packages/stylist/src/yew/mod.rs

```diff
@@ -3,7 +3,11 @@
 use yew::html::Classes;
 use yew::html::IntoPropValue;
 
-/// A procedural macro to style Yew component.
+/// A procedural macro to style a function component. Specifically this introduces a
```

Comment:
Better to move the sentence after the fist full stop to a new paragraph? So it looks nicer in the documentation of `stylist::yew`.